plugins/package-managers/node/src/main/kotlin/Npm.kt

```diff
@@ -335,7 +335,7 @@ open class Npm(
             }
         }
 
-        downloadUrl = fixNpmDownloadUrl(downloadUrl)
+        downloadUrl = downloadUrl.fixNpmDownloadUrl()
```

Comment:
Here, `downloadUrl` still is a `var`, so the commit message is a bit misleading.

Reply:
I've reworded it again.